assets/js/bundle.js

```diff
@@ -25,7 +26,7 @@
 	/**
 	 * Multilevel accordion menu for mobile
 	 */
-	jQuery("li").each(function() {
+	jQuery("li").each(function () {
```

Comment:
🚫 **[phpcs]** <PEAR.Functions.FunctionCallSignature.ContentAfterOpenBracket> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Opening parenthesis of a multi-line function call must be the last content on the line